### ISSUE TITLE
Targeted Experiments

### ISSUE THREAD

**Issue by msakarvadia** [msakarvadia/topology_aware_learning#46]
A set of experiments that will help us discern the direction we want to take:

- [x] Random weighting w/ softmax in 3 levels of data heterogeneity
- [x] Create figure for the weighted experiments that inspired the softmax (the heterogeneous ... homogeneous experiments)
- [x] aggregate w/ softmax across 3 levels of data heterogeneity
- [x] CIFAR 10 
- [x] 3 rounds of increasing connectivity
- [x] 3 rounds of increasing learning rate in a setting we already know works (later experiment)
- [ ] experiment w/ 3 different epochs counts between aggregation (maybe this is an extension)
- [x] Drop backdoor in 3 different locations in a single topology w/ a learning strategy we know that works (later experiment)
